manifests/mailer.pp

```diff
@@ -16,6 +29,7 @@
       teams => $teams,
     }
   }
+
```

Comment:
can you chain order this so we don't deploy the monitoring before we deploy the handler (and therefore the dependencies?) All it needs is a `->`